metadata_service/config.py

```diff
@@ -34,6 +34,6 @@ class LocalConfig(Config):
     LOG_LEVEL = 'DEBUG'
     LOCAL_HOST = '0.0.0.0'
 
-    PROXY_HOST = f'bolt://{LOCAL_HOST}'
-    PROXY_PORT = 7687
+    PROXY_HOST = os.environ.get('PROXY_HOST', f'bolt://{LOCAL_HOST}')
+    PROXY_PORT = os.environ.get('PROXY_PORT', 7687)
     PROXY_CLIENT = PROXY_CLIENTS['NEO4J']
```

Comment:
Sweet! Maybe worth mentioning the use of OS environment vars for HOST and PORT together with `PROXY_CLIENT` in https://github.com/lyft/amundsenmetadatalibrary/blob/master/docs/structure.md#proxy-package?

For consistency it would also be nice to add similar to https://github.com/lyft/amundsensearchlibrary and depend on it in https://github.com/lyft/amundsenfrontendlibrary/blob/master/docker-amundsen.yml

Reply:
@jornh good point. Updated docs https://github.com/lyft/amundsenmetadatalibrary/pull/27/commits/20e3992fda839b4ab0d055acdc138cabb68fa6ce